src/types.ts

```diff
@@ -62,6 +62,14 @@ export interface Caches {
    * List of the different building locations a class can be at
    */
   locations: Location[];
+  /**
+   * List of the all the dates on which finals are happening
+   */
+  finalDates: Date[];
+  /**
+   * List of the time blocks for finals
+   */
+  finalTimes: string[];
```

Comment:
Grabbing latest docs from https://github.com/gt-scheduler/website/pull/134 to keep them in sync.

```suggestion
  /**
   * List of the all the dates on which finals are happening
   * Example date: Aug 02, 2022
   */
  finalDates: Date[];
  /**
   * List of the time blocks for finals
   * Example time: 11:20 am - 2:10 pm
   */
  finalTimes: string[];
```